lib/web/apiserver.go

```diff
@@ -2328,3 +2315,39 @@ func makeTeleportClientConfig(ctx *SessionContext) (*client.Config, error) {
 
 	return config, nil
 }
+
+func redirectToErrorCallbackPage(w http.ResponseWriter, r *http.Request) (interface{}, error) {
+	pathToError := url.URL{Path: "/web/msg/error/login_failed_callback"}
+	http.Redirect(w, r, pathToError.String(), http.StatusFound)
+
+	return nil, nil
+}
+
+func redirectToErrorPage(w http.ResponseWriter, r *http.Request) (interface{}, error) {
```

Comment:
revised, i created this helper function to help me do one liner return statements `return redirectToErrorPage`, instead of doing two liner like below since redirecting didn't stop code execution:
```
http.Redirect(w, r, loginFailedURL, http.Status.Found)
return nil, nil
```